Bottom: fe423c6df99ecc009ac9a702d026b78c8461ef69
Top:    b9821977dcb40485583408d46e1ac8a95854c311
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 14:12:52 -0500

Refresh of wiki-remove-libicu52

---
diff --git a/docker/docker-mantisbt/Dockerfile b/docker/docker-mantisbt/Dockerfile
index 7db874b3..b59a97c7 100644
--- a/docker/docker-mantisbt/Dockerfile
+++ b/docker/docker-mantisbt/Dockerfile
@@ -8,22 +8,23 @@ MAINTAINER Joseph Lutz <Joseph.Lutz@novatechweb.com>
 
 ENV MANTISBT_VERSION 1.2.19
 
-#         ttf-mscorefonts-installer
-
 RUN sed -i 's| main$| main contrib non-free|' /etc/apt/sources.list \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install --yes --no-install-recommends \
         libcurl3 \
+        libicu52 \
         libjpeg62-turbo \
         libmcrypt4 \
         libpq5 \
+        ttf-mscorefonts-installer \
+        \
         libcurl4-openssl-dev \
         libfreetype6-dev \
         libicu-dev \
         libjpeg62-turbo-dev \
         libldap2-dev \
         libmcrypt-dev \
-        libpng-dev \
+        libpng12-dev \
         libpq-dev \
     && docker-php-ext-configure ldap --with-libdir=lib/x86_64-linux-gnu \
     && docker-php-ext-configure gd --with-png-dir=/usr --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
